<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CylinderPropertiesDialog</class>
 <widget class="QDialog" name="CylinderPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CylinderPropertiesDialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Base radius:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtHeight">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtBaseRadius">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtColor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
